### DISCUSSION TITLE
create and send message to team's chat

### DISCUSSION THREAD

**Discussion by gregu81** [O365/python-o365#854] [Q&A]
hi,

First of all, really thanks for package.
I have one question, could You please share with some examples related to Teams.  I am interested in case to create chat and send some short message, but I have only mails, mine and second person. Is such scenario currently supported and how to do it?
thanks in advance,
BR,
Greg

**Comment by alejcas** [comment]
I don't use the Teams api so I can't help you there

**Reply by gregu81** [reply]
ok, thanks

**Reply by erickmaga** [reply]
@gregu81 
Did you ever figure out how to do it?
I have the same question.
